frontend/package.json

```diff
@@ -26,6 +26,7 @@
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.1",
     "react-simple-card": "^2.0.2",
+    "react-toastify": "^7.0.3",
```

Comment:
please don't install as new package for toast as we already have a toast component under `util` folder, use that.